test/runtests.jl

```diff
@@ -22,8 +22,8 @@ using Pkg.Artifacts
     Max was so happy that he had made a new friend. He and the cat played together all day and had lots of fun."""
     @test String(take!(buf)) == output
 
-    # Try without mmapping, to make sure it still works
```

Comment:
It would be better to run this as two different tests with and without mmap, since the first test actually runs a different code path. I don't think it would add too much overhead to have 3 tests in total

Reply:
We do have 3 tests, the two original tests, plus my new one which is the 2nd original test but w/ mmap on. I had an intermediate commit here though where I had mmap on by default, so I had switched this 3rd test to mmap off.